src/Web/Sfa.Das.Sas.Web.UnitTests/Infrastructure/Web/Services/Mapping/ForProviderMappingHelper.cs

```diff
@@ -42,18 +42,75 @@ public void WhenGetCommaList()
             ProviderMappingHelper.GetCommaList(list).Should().BeEquivalentTo(expected);
         }
 
-        [TestCase("day release", "DayRelease")]
-        [TestCase("block release", "BlockRelease")]
-        [TestCase("at your location", "100PercentEmployer")]
-        [TestCase("day release, block release", "DayRelease", "BlockRelease")]
-        [TestCase("day release, at your location", "DayRelease", "100PercentEmployer")]
-        [TestCase("block release, at your location", "BlockRelease", "100PercentEmployer")]
-        [TestCase("day release, block release", "BlockRelease", "DayRelease")]
-        [TestCase("day release, at your location", "100PercentEmployer", "DayRelease")]
-        [TestCase("block release, at your location", "100PercentEmployer", "BlockRelease")]
-        public void WhenGetDeliveryOptionTextt(string expected, params string[] input)
+        [TestCase(
```

Comment:
The test cases are a little difficult to read. Could they be rationalised into a testcasesource and written out using string interpolation?